2048/2048/src/main/java/com/appcoins/eskills2048/MainGame.java

```diff
@@ -290,10 +305,10 @@ private void checkLose() {
   }
 
   private void endGame() {
-    endGame(true);
+    endGame(true,false);
```

Comment:
this is minor, but i noticed that there are some classes where codestyle is not properly applied. if it helps, you can commit through the IDE and select `Reformat code` and `Optimize imports` to be applied automatically before the commit.

Reply:
Thanks for the tip, already have it set up